src/main/java/com/minecolonies/coremod/entity/ai/citizen/guard/DruidCombatAI.java

```diff
@@ -0,0 +1,340 @@
+package com.minecolonies.coremod.entity.ai.citizen.guard;
+
+import com.google.common.collect.ImmutableList;
+import com.minecolonies.api.colony.guardtype.registry.ModGuardTypes;
+import com.minecolonies.api.entity.ai.statemachine.tickratestatemachine.ITickRateStateMachine;
+import com.minecolonies.api.entity.citizen.Skill;
+import com.minecolonies.api.entity.combat.threat.IThreatTableEntity;
+import com.minecolonies.api.entity.pathfinding.PathResult;
+import com.minecolonies.api.entity.pathfinding.PathingOptions;
+import com.minecolonies.api.items.ModItems;
+import com.minecolonies.api.util.BlockPosUtil;
+import com.minecolonies.api.util.InventoryUtils;
+import com.minecolonies.coremod.colony.buildings.AbstractBuildingGuards;
+import com.minecolonies.coremod.colony.buildings.modules.settings.GuardTaskSetting;
+import com.minecolonies.coremod.colony.jobs.AbstractJobGuard;
+import com.minecolonies.coremod.colony.jobs.JobDruid;
+import com.minecolonies.coremod.entity.DruidPotionEntity;
+import com.minecolonies.coremod.entity.ai.combat.AttackMoveAI;
+import com.minecolonies.coremod.entity.citizen.EntityCitizen;
+import com.minecolonies.coremod.entity.pathfinding.MinecoloniesAdvancedPathNavigate;
+import com.minecolonies.coremod.entity.pathfinding.pathjobs.AbstractPathJob;
+import com.minecolonies.coremod.entity.pathfinding.pathjobs.PathJobCanSee;
+import com.minecolonies.coremod.entity.pathfinding.pathjobs.PathJobMoveAwayFromLocation;
+import com.minecolonies.coremod.entity.pathfinding.pathjobs.PathJobMoveToLocation;
+import net.minecraft.entity.LivingEntity;
+import net.minecraft.entity.ai.attributes.Attributes;
+import net.minecraft.entity.player.PlayerEntity;
+import net.minecraft.item.ItemStack;
+import net.minecraft.item.Items;
+import net.minecraft.item.PotionItem;
+import net.minecraft.potion.*;
+import net.minecraft.util.Hand;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.function.BiPredicate;
+
+import static com.minecolonies.api.research.util.ResearchConstants.DRUID_USE_POTIONS;
+import static com.minecolonies.api.util.constant.GuardConstants.*;
+import static com.minecolonies.coremod.entity.ai.basic.AbstractEntityAIFight.SPEED_LEVEL_BONUS;
+
+/**
+ * Druid combat AI
+ */
+public class DruidCombatAI extends AttackMoveAI<EntityCitizen>
+{
+    /**
+     * List of potential positive effects.
+     */
+    private static final ImmutableList<Effect> SUPPORT_EFFECTS = ImmutableList.of(Effects.DAMAGE_BOOST, Effects.SATURATION, Effects.HEAL, Effects.DAMAGE_RESISTANCE);
+
+    /**
+     * List of potential positive effects.
+     */
+    private static final ImmutableList<Effect> ADVERSE_EFFECTS = ImmutableList.of(Effects.MOVEMENT_SLOWDOWN, Effects.WEAKNESS);
+
+    /**
+     * The xp per thrown potion
+     */
+    private static final double PER_POTION_XP = 0.05D;
```

Comment:
Does he not get the usual guard kill XP? how does it compare to the other guards XP gain?

Reply:
They don't ever seem to kill mobs, just weaken them

Reply:
They are quite unlikely to kill, they regularly switch targets too, so that seemed like the best option.